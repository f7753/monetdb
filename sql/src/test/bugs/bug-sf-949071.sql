select name from tables, (select count(*) from tables where "istable" = true) as t2 where "istable" = true and "system" = true;
select name from (select count(*) from tables where "istable" = true) as t2, tables where "istable" = true and "system" = true;
