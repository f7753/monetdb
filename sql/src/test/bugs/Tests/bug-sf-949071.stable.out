stdout of test 'bug-sf-949071` in directory 'src/test/bugs` itself:


# 19:25:26 >  
# 19:25:26 >  Mtimeout -timeout 180 Mserver "--config=/ufs/niels/data/Linux-i686/etc/monet.conf" --debug=10 --set "monet_mod_path=/ufs/niels/data/Linux-i686/lib/MonetDB" --set "gdk_dbfarm=/ufs/niels/data/Linux-i686/var/MonetDB/dbfarm" --set "sql_logdir=/ufs/niels/data/Linux-i686/var/MonetDB/log" --set mapi_port=46563 --set sql_port=55013 --set monet_prompt= --trace "--dbname=mTests_src_test_bugs" "/net/stem.ins.cwi.nl/export/scratch1/niels/Linux-i686/build-sql/src/backends/monet4/sql_server.mil" ; echo ; echo Over..
# 19:25:26 >  

# Monet Database Server V4.3.17
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# Visit http://monetdb.cwi.nl for further information.

printf("\nReady.\n");

Ready.
quit();

Over..

# 01:20:33 >  
# 01:20:33 >  Mtimeout -timeout 60 MapiClient --language=sql -u monetdb -P monetdb --host=stem --port=54544  < bug-sf-949071.sql
# 01:20:33 >  

# 3 # querytype
# tables,	tables,	tables,	tables,	tables,	tables,	tables,	tables,	t2 # table_name
# id,	name,	schema_id,	query,	istable,	system,	clear,	temporary,	count_id # name
# mediumint,	varchar,	mediumint,	varchar,	boolean,	boolean,	boolean,	decimal,	mediumint # type
# 0,	10,	0,	0,	0,	0,	0,	0,	0 # length
# 29,	,	,	,	,	,	,	,	 # tuplecount
# 1,	,	,	,	,	,	,	,	 # id
[ 1005868,	"modules",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1005919,	"schemas",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1005923,	"ptables",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1005931,	"pcolumns",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1005941,	"keys",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1005947,	"idxs",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1005952,	"keycolumns",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1005957,	"types",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1005965,	"ttables",	1005858,	"",	true,	true,	true,	0,	29	]
[ 1006003,	"tcolumns",	1005858,	"",	true,	true,	true,	0,	29	]
[ 1006051,	"db_users",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1006074,	"user_role",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1006097,	"auths",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1006115,	"privileges",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1006143,	"history",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1006181,	"sessions",	1005858,	"",	true,	true,	true,	0,	29	]
[ 1006204,	"env",	1005858,	"",	true,	true,	false,	0,	29	]
[ 1009330,	"foo",	1005858,	"",	true,	false,	false,	0,	29	]
[ 1009468,	"ff",	1005858,	"",	true,	false,	false,	0,	29	]
[ 1009946,	"s4",	1005858,	"",	true,	false,	false,	0,	29	]
[ 1011787,	"r",	1005858,	"",	true,	false,	false,	0,	29	]
[ 1006273,	"test",	1005858,	"",	true,	false,	false,	0,	29	]
[ 1006294,	"c1006293",	1005858,	"",	true,	false,	false,	1,	29	]
[ 1006338,	"c1006337",	1005858,	"",	true,	false,	false,	1,	29	]
[ 1006381,	"tables",	1005858,	"",	true,	false,	false,	1,	29	]
[ 1006425,	"c1006424",	1005858,	"",	true,	false,	false,	1,	29	]
[ 1006469,	"c1006468",	1005858,	"",	true,	false,	false,	1,	29	]
[ 1006512,	"tables",	1005858,	"",	true,	false,	false,	1,	29	]
[ 1006555,	"t2",	1005858,	"",	true,	false,	false,	1,	29	]
# 3 # querytype
# t2,	tables,	tables,	tables,	tables,	tables,	tables,	tables,	tables # table_name
# count_id,	id,	name,	schema_id,	query,	istable,	system,	clear,	temporary # name
# mediumint,	mediumint,	varchar,	mediumint,	varchar,	boolean,	boolean,	boolean,	decimal # type
# 0,	0,	10,	0,	0,	0,	0,	0,	0 # length
# 36,	,	,	,	,	,	,	,	 # tuplecount
# 2,	,	,	,	,	,	,	,	 # id
[ 36,	1005868,	"modules",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1005919,	"schemas",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1005923,	"ptables",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1005931,	"pcolumns",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1005941,	"keys",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1005947,	"idxs",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1005952,	"keycolumns",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1005957,	"types",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1005965,	"ttables",	1005858,	"",	true,	true,	true,	0	]
[ 36,	1006003,	"tcolumns",	1005858,	"",	true,	true,	true,	0	]
[ 36,	1006051,	"db_users",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1006074,	"user_role",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1006097,	"auths",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1006115,	"privileges",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1006143,	"history",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1006181,	"sessions",	1005858,	"",	true,	true,	true,	0	]
[ 36,	1006204,	"env",	1005858,	"",	true,	true,	false,	0	]
[ 36,	1009330,	"foo",	1005858,	"",	true,	false,	false,	0	]
[ 36,	1009468,	"ff",	1005858,	"",	true,	false,	false,	0	]
[ 36,	1009946,	"s4",	1005858,	"",	true,	false,	false,	0	]
[ 36,	1011787,	"r",	1005858,	"",	true,	false,	false,	0	]
[ 36,	1006273,	"test",	1005858,	"",	true,	false,	false,	0	]
[ 36,	1006294,	"c1006293",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006338,	"c1006337",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006381,	"tables",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006425,	"c1006424",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006469,	"c1006468",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006512,	"tables",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006555,	"t2",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006840,	"c1006839",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006884,	"c1006883",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006927,	"tables",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006970,	"t2",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1006979,	"c1006978",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1007023,	"c1007022",	1005858,	"",	true,	false,	false,	1	]
[ 36,	1007066,	"tables",	1005858,	"",	true,	false,	false,	1	]

# 20:38:54 >  
# 20:38:54 >  Done.
# 20:38:54 >  

